Assets/MixedRealityToolkit.SDK/Features/Input/InputSimulationIndicators.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Input
```

Comment:
Hm...shouldn't this code live near where the input simulation files are?

Reply:
The indicators are an optional asset that is defined by the profile (default profile also lives in the SDK).

I'm not sure what the policy is regarding optional assets tied to services. It's just looks to me like we put all default assets in SDK, only have scripts in Services.